l10n_br_account/models/account_invoice.py

```diff
@@ -246,7 +246,7 @@ def unlink(self):
     def copy(self, default=None):
         default = default or {}
         if self.document_type_id:
-            default["line_ids"] = False
+            default["invoice_line_ids"] = False
```

Comment:
```suggestion
            default["fiscal_line_ids"] = False
```
@marcelsavegnago  não sei se testou enganado, não testei o código, mas na 12.0 o line_ids era o campo que é puxado do documento fiscal, hoje na 14.0 essa campo se chama fiscal_line_ids.

Reply:
ahh blzzz.. tem razao.

Reply:
Done